### DISCUSSION TITLE
Have nnn use the colors in my Xresources

### DISCUSSION THREAD

**Discussion by em-777** [jarun/nnn#1795] [Ideas]
I love nnn, but one thing that annoys me is that it doesn't respect my Xresources. The default blue, green, and red clash horribly with the custom colors used by the rest of my system. I know suckless programs like DWM and ST have patches that allow them to read from your Xresources file and use the colors defined within. Can we do something similar with nnn? Thanks.

**Comment by N-R-K** [comment]
> The default blue, green, and red clash horribly with the custom colors used by the rest of my system.

See `NNN_FCOLORS` section on the manpage. And if you're using icons, you'll have to customize colors via editing `src/icons.h`.

> that it doesn't respect my Xresources.

`nnn` is a terminal file-manager. Not an X11 GUI.

**Reply by em-777** [reply]
> See NNN_FCOLORS section on the manpage.

Eh, this will have to do.
